aspnetcore/security/authentication/disable-cookie.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Disable automatic cookie authentication in ASP.NET Core
+author: John0King
+description: Disable automatic cookie authentication in ASP.NET Core
+ms.author: riande
+ms.date: 5/22/2019
+uid: security/authentication/disable-cookie
+---
+# Disable automatic cookie authentication in ASP.NET Core
+
+By [John King](https://github.com/John0King) and [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+The [`[Authorize]`](xref:Microsoft.AspNetCore.Authorization.AuthorizeAttribute) specifies that access to a controller or action method is restricted to users who meet the authorization requirement. When the user is not authenticated or doesn't have access to the controller or action method:
+
+* An automatic challenge is issued, redirecting the user to the sign-in page.
+
+This approach works well when using ASP.NET Controllers with views or Razor Pages, but not when using [AJAX](https://developer.mozilla.org/en-US/docs/Web/Guide/AJAX). This document shows several approaches to take when a HTTP StatusCode 401 response is required for unauthenticated/unauthorized requests.
+
+Use one of the following approaches to disable automatic cookie authentication:
```

Comment:
It's not the automatic authentication that's being disabled, it's the automatic *challenge* that ends up in a response redirect. I don't think challenging is described in the docs, so it might need a different explanation, but I'd read "automatic cookie authentication" as meaning the process of automatically turning a cookie from the request into a `ClaimsPrincipal`.

Reply:
I changed it to 
`Use one of the following approaches to disable automatic cookie challenge:`

Reply:
Yeah, I think that makes sense. The linked issue talks of replacing `AutomaticChallenge` so I think the terminology is good. It looks like it might need to be updated elsewhere in the page too from what I can see.